src/components/picker/types.tsx

```diff
@@ -147,49 +130,112 @@ export type PickerBaseProps = Omit<NewTextFieldProps, 'value' | 'onChange'> & {
    * Render custom search input (only when passing showSearch)
    */
   renderCustomSearch?: (props: PickerItemsListProps) => React.ReactElement;
+};
+
+type PickerExpandableOverlayProps = {
   /**
-   * Render a custom header for Picker's dialog
+   * Custom picker props (when using renderPicker, will apply on the button wrapper)
    */
-  renderCustomDialogHeader?: (callbacks: {onDone?: () => void, onCancel?: ()=> void}) => React.ReactElement;
+  customPickerProps?: ExpandableOverlayProps;
   /**
-   * Use wheel picker instead of a list picker
+   * Render custom picker modal (e.g ({visible, children, toggleModal}) => {...})
    */
-  useWheelPicker?: boolean;
+  renderPickerOverlay?: (modalProps: RenderCustomModalProps) => React.ReactElement;
```

Comment:
Why not simply 'renderOverlay' like 'renderInput'?

Reply:
renamed